pkg/kubelet/kubeletconfig/configfiles/configfiles_test.go

```diff
@@ -149,7 +157,9 @@ staticPodPath: %s`, relativePath)),
 			}
 			kc, err := loader.Load()
 			if utiltest.SkipRest(t, c.desc, err, c.err) {
-				return
+				if c.strictErr && !runtime.IsStrictDecodingError(errors.Cause(err)) {
```

Comment:
I can't quite remember what I was thinking at the time, but the way `SkipRest` is written (specifically, logging the name of the test case), makes me wonder if I implemented it before I started to use subtests in most places. Looking at it now, it seems kind of silly to have to return a bool rather than just use `t.Fatalf`. 

Not that we should try to fix it in this PR, as it's used in a lot of places, but it may be simpler to just do the `c.strictErr && !runtime.IsStrictDecodingError(errors.Cause(err))` test before the `SkipRest` call, and just call `t.Fatalf` if it fails, rather than conditioning the strict decoding error test on the return value of `SkipRest`.

Reply:
Also wanted to ask why we need to use `errors.Cause`? We should probably be using the new built-in error wrapping functionality available in Go 1.13, instead of `github.com/pkg/errors`.

Do we need to change `IsStrictDecodingError` to use `errors.As`? https://github.com/golang/go/wiki/ErrorValueFAQ

Reply:
I moved the check before `SkipRest` (which is now just returning again). I also moved from `errors.Cause` to `%w` formatting, using `errors.Unwrap` from the stdlib in the test.

Reply:
Ah, didn't realize we hadn't adopted 1.13 yet. 
I think %w came with 1.13 too, so we'll probably have to wait for that too.

Reply:
Alright. How should we proceed with this patch? Use `errors.Wrap` for time being?